testproto/ignore_unknown_fields/opt_vtproto.pb.go

```diff
@@ -0,0 +1,307 @@
+// Code generated by protoc-gen-go-vtproto. DO NOT EDIT.
+// protoc-gen-go-vtproto version: (devel)
+// source: ignore_unknown_fields/opt.proto
+
+package ignore_unknown_fields
+
+import (
+	fmt "fmt"
+	protohelpers "github.com/planetscale/vtprotobuf/protohelpers"
+	proto "google.golang.org/protobuf/proto"
+	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
+	io "io"
+	unsafe "unsafe"
+)
+
+const (
+	// Verify that this generated code is sufficiently up-to-date.
+	_ = protoimpl.EnforceVersion(20 - protoimpl.MinVersion)
+	// Verify that runtime/protoimpl is sufficiently up-to-date.
+	_ = protoimpl.EnforceVersion(protoimpl.MaxVersion - 20)
+)
+
+func (m *IgnoreUnknownFieldsExtension) CloneVT() *IgnoreUnknownFieldsExtension {
+	if m == nil {
+		return (*IgnoreUnknownFieldsExtension)(nil)
+	}
+	r := new(IgnoreUnknownFieldsExtension)
+	r.Foo = m.Foo
+	return r
+}
+
+func (m *IgnoreUnknownFieldsExtension) CloneMessageVT() proto.Message {
+	return m.CloneVT()
+}
+
+func (this *IgnoreUnknownFieldsExtension) EqualVT(that *IgnoreUnknownFieldsExtension) bool {
+	if this == that {
+		return true
+	} else if this == nil || that == nil {
+		return false
+	}
+	if this.Foo != that.Foo {
+		return false
+	}
+	return string(this.unknownFields) == string(that.unknownFields)
```

Comment:
Sorry! You missed this one! Please give a quick review to the all the codegen steps to ensure that nothing else is using `unknownFields`. A `git grep` for `*.go` files will find everything. :)

Reply:
I've changed it.